Mods/ArchipelagoTriggers.SC2Mod/Base.SC2Data/LibABFE498B.galaxy

```diff
@@ -7194,6 +7194,12 @@ void libABFE498B_gf_AP_Triggers_Protoss_unlockReaverResourceEfficiency (int lp_p
     libNtve_gf_SetUpgradeLevelForPlayer(lp_player, "AP_ResourceEfficiencyReaver", 1);
 }
 
+void libABFE498B_gf_AP_Triggers_Protoss_unlockReaverKhalaiReplicators (int lp_player) {
+    // Automatic Variable Declarations
+    // Implementation
+    libNtve_gf_SetUpgradeLevelForPlayer(lp_player, "AP_ReaverKhalaiReplicators", 1);

```

Comment:
Turn on autocast for all the player's Reavers. This is done with L2 replenishable mines for Vultures (otherwise Autocast should be off by default as it costs resources)

Reply:
Unsure this is the right decision, as Reavers can't attack without scarabs so players will basically always want this on. Will bring it up in the discord.

Reply:
Threw up a poll in #test-games, current votes are 6-0 in favour of autocast by default. Can wait a few more hours to see if more votes come it, but it seems pretty clear to me players prefer autocast on by default.

Reply:
Poll closed 10-0 in favour of leaving autocast on. Hence I'm not changing this.